chef-utils/lib/chef-utils/dsl.rb

```diff
@@ -0,0 +1,30 @@
+
+# NOTE: NOT EVERYTHING GOES IN HERE!  THIS GETS INJECTED EVERYWHERE!
+#
+# - ChefUtils::Service is not intended to be injected into the DSL, do not add it.
+#
+# - Any other helpers should NOT be added here until they are proven.
+#
+
+require_relative "internal"
+require_relative "architecture"
+require_relative "os"
+require_relative "platform_family"
+require_relative "platform"
+require_relative "introspection"
+require_relative "which"
+require_relative "path_sanity"
+
+module ChefUtils
+  module DSL
+    include ChefUtils::Architecture
+    include ChefUtils::OS
+    include ChefUtils::PlatformFamily
+    include ChefUtils::Platform
+    include ChefUtils::Introspection
+    include ChefUtils::Internal
+    # FIXME: include ChefUtils::Which in Chef 16.0
```

Comment:
looking forward to these in ohai 16

Reply:
we can probably pull them into ohai 15

Reply:
yeah I guess it doesn't block that work being done. It would be nice to kill off the similar but different which logic there.

Reply:
(ohai can define its own DSL)